src/main/java/common/DriverBinariesSetter.java

```diff
@@ -14,23 +14,31 @@
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.security.GeneralSecurityException;
+import java.util.Base64;
 
 import static common.DriverBinariesSetter.Drivers.*;
 
 
 public final class DriverBinariesSetter {
 
-    private  static DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+    private static final DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+    private static final String sauceLabUserName =
+            new String(Base64.getDecoder().decode(ConfigurationManager.getProperty("sauceLab.username")));
+    private static final String sauceLabAccessKey =
```

Comment:
I'm not sure it's a real encoding of sensitive data, it's more similar to converting Base64 -> Credential values. Please implement any algorithm using a secret key